public_html/storefront/controller/pages/content/content.php

```diff
@@ -26,59 +26,247 @@ class ControllerPagesContentContent extends AController
     public function main()
     {
         $request = $this->request->get;
-
+        $this->data = [];
         //init controller data
         $this->extensions->hk_InitData($this, __FUNCTION__);
 
         $this->loadModel('catalog/content');
-
         $this->document->resetBreadcrumbs();
-
         $this->document->addBreadcrumb(array(
             'href'      => $this->html->getHomeURL(),
             'text'      => $this->language->get('text_home'),
             'separator' => false,
         ));
 
-        if (isset($request['content_id'])) {
-            $content_id = $request['content_id'];
-        } else {
-            $content_id = 0;
-        }
+        $content_id = isset($request['content_id']) ? $request['content_id'] : 0;
+        $page = isset($request['page']) ? $request['page'] : 1;
```

Comment:
FYI, you can write shorter here
$page = $request['page'] ?: 1;  // page cannot be zero in any case
or 
$sort = $request['sort'] ?? 'default';